tests/vmi_configuration_test.go

```diff
@@ -234,6 +235,94 @@ var _ = Describe("Configurations", func() {
 		})
 	})
 
+	Context("with CPU spec", func() {
+		cpuRegexp := regexp.MustCompile(`<model>(\w+)\-*\w*</model>`)
+		vendorRegexp := regexp.MustCompile(`<vendor>(\w+)</vendor>`)
+
+		var cpuModel string
+		var cpuVendor string
+		var cpuVmi *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			nodes, err := virtClient.CoreV1().Nodes().List(metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(nodes.Items).NotTo(BeEmpty())
+
+			virshCaps := tests.GetNodeLibvirtCapabilities(nodes.Items[0].Name)
+
+			model := cpuRegexp.FindStringSubmatch(virshCaps)
+			Expect(len(model)).To(Equal(2))
+			cpuModel = model[1]
+
+			vendor := vendorRegexp.FindStringSubmatch(virshCaps)
+			Expect(len(vendor)).To(Equal(2))
+			cpuVendor = vendor[1]
+
+			cpuVmi = tests.NewRandomVMIWithEphemeralDiskAndUserdata(tests.RegistryDiskFor(tests.RegistryDiskCirros), "#!/bin/bash\necho 'hello'\n")
+			cpuVmi.Spec.Affinity = &v1.Affinity{
+				NodeAffinity: &kubev1.NodeAffinity{
+					RequiredDuringSchedulingIgnoredDuringExecution: &kubev1.NodeSelector{
+						NodeSelectorTerms: []kubev1.NodeSelectorTerm{
+							{
+								MatchExpressions: []kubev1.NodeSelectorRequirement{
+									{Key: "kubernetes.io/hostname", Operator: kubev1.NodeSelectorOpIn, Values: []string{nodes.Items[0].Name}},
+								},
+							},
+						},
+					},
+				},
+			}
+		})
+
+		Context("when CPU model definied", func() {
```

Comment:
defined, here and below

Reply:
@booxter below I have `Context("when CPU model not definied"`

Reply:
I meant that "definied" -> "defined".

Reply:
oh will create small PR for it